docker-compose.yml

```diff
@@ -5,14 +5,25 @@ version: "3.9"
 services:
   auto-gpt:
     depends_on:
-      - redis
+      - redisStack
     build: ./
     env_file:
       - .env
     volumes:
-      - "./autogpt:/home/appuser"
-      - ".env:/home/appuser/.env"
+      - "./autogpt:/app"
+      - ".env:/app/.env"
+      - "$PWD/auto_gpt_workspace:/home/appuser/auto_gpt_workspace"
+      - "./ai_settings.yaml:/home/appuser/ai_settings.yaml"
```

Comment:
This is a good idea, but if you dont have a file there to start with, docker compose creates a dir, and you get hit with the following:
```
  File "/home/appuser/autogpt/cli.py", line 126, in main
    system_prompt = construct_prompt()
  File "/home/appuser/autogpt/prompt.py", line 173, in construct_prompt
    config = AIConfig.load(CFG.ai_settings_file)
  File "/home/appuser/autogpt/config/ai_config.py", line 61, in load
    with open(config_file, encoding="utf-8") as file:
IsADirectoryError: [Errno 21] Is a directory: 'ai_settings.yaml'
```
It would be good if this file was created in the repo, maybe even a template like env.template, and adding something to the readme instructions

Reply:
i am removing the ai_settings mount as it's not entirely necessary